app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(cp: str) -> None:
+    without_cp = cp.split()
+    file_1 = without_cp[1]
+    file_2 = without_cp[2]
+    if file_1 == file_2:
+        return
+    with open(file_1, "r") as file1:
+        with open(file_2, "w") as file2:
```

Comment:
<img width="1003" alt="image" src="https://github.com/mate-academy/py-copy-file/assets/65166203/5351ad8e-391f-4ed0-987f-7e3d4f7c532c">
Do like this